packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster-app-card.tsx

```diff
@@ -122,37 +131,64 @@ export const ClusterAppCard: React.FC = () => {
     namespace: undefined,
     name: ALL_APPS,
   });
-  const [pvcSLAData, pvcSLAError, pvcSLALoading] = useCustomPrometheusPoll({
-    endpoint: 'api/v1/query' as any,
-    query: !!cluster ? getPvcSlaPerClusterQuery(cluster) : null,
-    basePath: ACM_ENDPOINT,
+  const [mcvs, mcvsLoaded, mcvsLoadError] = useK8sWatchResource<
+    ACMManagedClusterViewKind[]
+  >({
+    kind: referenceForModel(ACMManagedClusterViewModel),
+    isList: true,
+    namespace: cluster,
+    namespaced: true,
+    optional: true,
```

Comment:
can u try this and check if it works ?
```suggestion
    kind: referenceForModel(ACMManagedClusterViewModel),
    isList: true,
    namespace: cluster,
    namespaced: true,
    optional: true,
    fieldSelector: spec.scope.resource = DRVolumeReplicationGroup.kind,
```

Reply:
no need to filter later...

Reply:
it wont fieldSelector won't work for all fields: https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/

Reply:
when i tried i got error, like unsupported field selector-error